shared/rakefiles/xk_config.rake

```diff
@@ -38,12 +50,15 @@ rule @kubectl_creds_file => [@gcp_creds_file] do
   # This duplicates information in terraform code, 'zone'. Could be a variable with some plumbing.
   zone = 'us-central1-a'
   sh "
-    existing_cluster=$(gcloud container clusters list --filter #{cluster_name} --zone #{zone} --project #{ENV["TF_VAR_project_id"]})
+    existing_cluster=$(gcloud container clusters list --filter #{cluster_name} --zone #{zone} --project $TF_VAR_project_id)
     if [ $? == 0 ] && [ \"$existing_cluster\" != \"\" ]; then \
-      gcloud container clusters get-credentials #{cluster_name} --zone #{zone} --project #{ENV["TF_VAR_project_id"]}
+      gcloud container clusters get-credentials #{cluster_name} --zone #{zone} --project $TF_VAR_project_id
     fi"
 end
 
-task :configure_current_project => [@gcp_creds_file] do
+task :configure => [@gcp_creds_file, @default_creds_file, @kubectl_creds_file] do
```

Comment:
How about `:configure_creds`?

Reply:
I would like to keep it as `:configure` and add more stuff (like [auth user email setting](https://github.com/gpii-ops/gpii-infra/pull/225/files#diff-9bf20f39183745233b6ee4d09596753fR70)) that requires to be executed on every run.